### PR TITLE
Edited UG for Expense Splitter 

### LINKED ISSUE #174: Update DG for ExpenseSpliiter ListReceipt, ListAmount and PaidCommand

### PR DESCRIPTION
**Pull request by WillySeahh**

### CODE REVIEW

File: docs/diagrams/ExpenseSplitterListReceiptCommandSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+entry --[MODEL_COLOR]> receipt : String
+deactivate entry
+
+
+receipt --[MODEL_COLOR]-> model : String
+deactivate receipt
+
+
+model --[LOGIC_COLOR]> logic : CommandResult
+deactivate model
```
Comment (sheexiong) [inline docs/diagrams/ExpenseSplitterListReceiptCommandSequenceDiagram.puml]: I think here got an extra empty line.

**Review by sheexiong — DISMISSED** [review]
LGTM

File: docs/diagrams/ExpenseSplitterListReceiptCommandSequenceDiagram.puml (hunk 1 above)
Comment (WillySeahh) [reply docs/diagrams/ExpenseSplitterListReceiptCommandSequenceDiagram.puml]: alright resolved!

**Review by sheexiong — APPROVED** [review]
Everything things look fine. LGTM

**Pull request merged**